pom.xml

```diff
@@ -4,7 +4,8 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.23</version>
+        <version>2.31-20170612.235547-1</version> <!-- TODO https://github.com/jenkinsci/plugin-pom/pull/67 -->
```

Comment:
Certainly I was not proposing a merge unless and until the dependency is merged and released. That just applies to any use of snapshots.

Really this PR is not intended to accomplish anything in particular except to demonstrate that the upstream PR is consumable.